games/game_manager.rb

```diff
@@ -0,0 +1,35 @@
+require 'json'
+require_relative './game'
+require_relative '../storage'
+
+class GamesManager
+  def initialize
+    @storage = Storage.new
+
+    @games = @storage.load_data('games')
+  end
+
+  def list_all_games
+    puts @games
+    puts 'This is all the games we have : '
+    @games.each_with_index do |game, index|
+      puts "#{index + 1}. multiplayer: #{game.multiplayer} - last_played: #{game.last_played_at}"
+    end
+    puts ' '
+  end
+
+  def add_game
+    puts 'Adding new game'
+    print 'publish_date yyyy-mm-dd: '
+    publish_date = gets.chomp
+    print 'multiplayer : '
```

Comment:
It will be nice to give a user guidance on whether to answer Yes/No or Y/N on the question of multiplayer to make the system user friendly.

![image](https://user-images.githubusercontent.com/41517876/219124953-39e8560e-4374-43f1-ba9d-b51f7f0bfef5.png)

Reply:
Good idea to improve user experience, i will da that.